acceptance/openstack/apigw/v2/env_vars_test.go

```diff
@@ -0,0 +1,66 @@
+package v2
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/apigw/v2/env_vars"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestEnvVarsLifecycle(t *testing.T) {
+	gatewayID := clients.EnvOS.GetEnv("GATEWAY_ID")
+	groupID := clients.EnvOS.GetEnv("GROUP_ID")
+	envID := clients.EnvOS.GetEnv("ENV_ID")
+
+	if gatewayID == "" || groupID == "" || envID == "" {
```

Comment:
how long does it take to create gw, group and env?

Reply:
yes